zipline/protocol.py

```diff
@@ -270,7 +272,7 @@ def __contains__(self, name):
     def __repr__(self):
         return "SIDData({0})".format(self.__dict__)
 
-    def _get_buffer(self, bars, field='price'):
+    def _get_buffer(self, bars, field='price', raw=False):
```

Comment:
Sure; however, using a private method is unsupported behaviour, people working with zipline should not be using this method in the first place. Attempting to backwards compat all private methods forever is too much effort for no reward.